### ISSUE TITLE
Add some benchmark expectations for running the examples

### ISSUE THREAD

**Issue by jchus** [choderalab/yank-examples#55]
Running the run-p-xylene-explicit.sh example, it took 2 days to minimize the 25 replicas and then it said:
"Estimated completion in 35 days, 14:08:37.851816"
Granted, it on a 2010 8-core MacPro (using platform=CPU) but I don't know if it is what to expect or if there is an issue with my setting. I'd prefer not to wait for 35 days to find out if the examples worked correctly.
It would be informative to have a (rough) idea of what the compute times for the examples should be (also informative to see the speedup between the different platforms).

**Comment by andrrizzi** [comment]
Good idea! You're right, unfortunately the examples are made to end in reasonable time only on GPU.

**Comment by jchodera** [comment]
We're very much only targeting GPUs at the moment, since OpenMM is heavily GPU-optimized but barely optimized for CPUs. Codes like [gromacs](http://gromacs.org), [NAMD](http://www.ks.uiuc.edu/Research/namd/), or [AMBER](http://ambermd.org/) are a much better choice for running on CPUs at the moment.

**Comment by jchodera** [comment]
Benchmarks are still a great idea, though!

We haven't done any optimization in YANK yet---there are still big inefficiencies---but we'll be sure to post some benchmarks when the full 1.0 release is ready!

**Comment by jchus** [comment]
Thank you for your feedback. I first tried the OpenCL platform but not much was happening (it seemed to be computing as the usage on the card was 12% but there was no output, so I thought to check that it could complete on the CPU platform). This was on a Radeon7970 though, so pretty slow. I'll try to get CUDA working on more recent Nvidia card I have but this has been a bit tricky on the Mac.

**Comment by jchodera** [comment]
We're also hoping to get some docker images that can be run on Amazon EC2 for those without easy access to local GPUs.
